releases/management/artifacts.md

```diff
@@ -0,0 +1,36 @@
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+# Published Artifacts
+
+The Splinter ecosystem includes a variety of published artifacts, including Rust
+crates and Docker images. All Splinter artifacts are versioned together; this
+means that for each release, a new version of each artifact will be published
+with the new release number. This allows developers to easily select compatible
+versions of all crates and images.
+
+Below is a list of all major artifacts that are published with every Splinter
+release.
+
+## Rust crates
+
+* [splinter](https://crates.io/crates/splinter)
+* [scabbard](https://crates.io/crates/scabbard)
```

Comment:
How about clarifying that these crates are for the splinter (aka libsplinter) and scabbard libraries?  It seems odd that the preceding test just talks about Splinter, but now there's splinter and scabbard.